js/src/components/paid-ads/ads-campaign.js

```diff
@@ -105,7 +119,7 @@ export default function AdsCampaign( {
 			<StepContentFooter>
 				<AppButton
 					isPrimary
-					disabled={ ! isValidForm }
+					disabled={ ! enableContinue }
```

Comment:
@kt-12 My understanding is that the Continue button should be disabled if:

1. the form is not valid (using the `isValidForm` variable)
2. if billing is not set up/approved.

How about we add the following condition then?
```js
disabled={
	! isValidForm ||
	billingStatus?.status !==
		GOOGLE_ADS_BILLING_STATUS.APPROVED
}
```

There is no need to show the spinner while the budget status is being resolved.

Reply:
Button will be disabled till we have status so we don't need the spinner, totally make sense to me. Thanks @asvinb .